### DISCUSSION TITLE
Various features wishlist, would love to discuss

### DISCUSSION THREAD

**Discussion by Riizade** [chilli-axe/mpc-autofill#249] [Ideas]
Hello!

I have a private application that does more or less the same thing as mpcfill.
I've been using it since before mpcfill existed.

I really love mpcfill, and I'd like to use it because its level of polish is much, much higher, but it's missing a few features my application has.

In particular,
- the ability to upload custom images to a project, if for example, you have a self-made proxy for a single card, or a custom back because you want, e.g., each of your cubes or decks to have its own card back so they're easily differentiated
- using Scryfall as an image source, so ~all official arts are available (at least in English)
- being able to change the card's back to any other card or art
    - allows you to have dual-faced cards for special foreign art, e.g., the Japanese Yoshitaka Amano Liliana, with the English version on the other side for playability reasons
    - also allows for filling blank mpcfill orders with double-sided tokens (e.g., a 540 card cube has 72 blank spaces in a 612 card order, allowing for 72 tokens with a different type of token on each face, for a potential 144 different token types; my tool uses Scryfall data to figure out which tokens the deck/cube can generate and automatically fills the blank spaces with dual-sided tokens)
    - dual-faced lands so you can have the same amount of land art choice with half the physical cards
- filter preferences that auto-select art by ranking your favorite sets, language preference, and full-art, borderless, etc, so that simply uploading a txt of just card names means 95% of the arts are already selected to be your preferred art, and you just need to double check and select the remaining 5%
- icon placement
    - I use this for placing a "sideboard" or "commander" icon on the corner of non-mainboard cards, so after playing I can easily swap sideboard cards out without having to remember or look up which cards were non-mainboard
    - also for custom jumpstart cubes, it lets me put an icon for each jumpstart pack on the corner, so you can separate the decks easily and play again

I can invite the mpc-autofill developers to the private repository to take a look around, and I'm available to discuss privately how I've implemented a bunch of these features

Let me know if any of these are things you think would be interesting for mpcfill and how you'd like to proceed discussions

**Comment by ndepaola** [comment]
howdy, thanks for reaching out! i'd love to organise a call to discuss this in detail. discord is easiest for me - my discord handle is chilli_axe - but can also do zoom / whatever other platform works for you

**Reply by Riizade** [reply]
Dope, I reached out via Discord, check the spam filter 'cuz we share servers but we're not registered as friends, so I think Discord places those messages in a separate inbox.

I'm @Riizade on Discord as well.

I also invited you to the private repo, feel free to look around and check it out.

**Reply by ndepaola** [reply]
sounds good! i'm at work atm (timezone is AEST) but i'll check discord + take a look at your repo after work.

**Comment by Cwazzle-afk** [comment]
I love mpc and I'd love it more if the search was optimized for finding unique art and easier ways to find the same artist's work through some sort of the tags that the sources could put on them. Also the source drives on the side aren't alphabetical and aren't searchable which makes finding previously used source drives difficult without the browsers search function. I'd like to use mpc as a way to find interesting art but it feels like digging through a bulk bin to find diamonds in the rough sometimes.

In short:

- Searchable drive sources 
- artist tags for finding similar art
- make a way to sort the arts based on popularity instead of upload date and alphabetical

Thank you so much for mpcfill
